src/common/iequals.cpp

```diff
@@ -0,0 +1,16 @@
+#include <cctype>
+#include <string>
+
```

Comment:
best to include the `iequals.h` header, such that we're sure they're consistent.